axelor-base/src/main/resources/views/Batch.xml

```diff
@@ -244,4 +248,12 @@
     <context name="_batch" expr="eval: __self__"/>
   </action-view>
 
+  <action-view name="action-view-import-histories" title="Import histories"
+    model="com.axelor.apps.base.db.BatchImportHistory">
+    <view type="grid" name="batch-import-history-grid"/>
+    <view type="form" name="batch-import-history-form"/>
+    <domain>:batch = self.batch</domain>
```

Comment:
it works this way ? shouldn't it be self.batch = :batch ?

Reply:
Indeed it does work